.editorconfig

```diff
@@ -0,0 +1,19 @@
+root = true
+
+[**]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+max_line_length = 80
+
+# Set default charset and indentation for python files
+[{*.py,coalib/**/*.py,./coala,tests/**/*.py}]
```

Comment:
@newbazz add spaces after `'` in line 10 and 16.

Reply:
no spaces to be added after `,` was discussed earlier

Reply:
Oh my bad.